.github/workflows/integration_testing.yml

```diff
@@ -80,3 +81,82 @@ jobs:
           for folder in integration-tests/*/; do
             (cd "$folder" && fastn test --headless)
           done
+
+  slides-integration-test:
```

Comment:
Why is this called slides test? Slides related tests should be in slides repo. Integration test in fastn should test fastn itself, the auth feature, endpoint using some example project that is part of this repo just for testing.